src/components/Alteration/summary/BusinessType.vue

```diff
@@ -45,6 +45,9 @@ import { ResourceUtilities } from '@/utils/resource-utils'
 export default class BusinessType extends Mixins(FilingTemplateMixin) {
   readonly GetCorpFullDescription = GetCorpFullDescription // for template
 
+  // Store getters
+  // @Getter(useStore) getOriginalLegalType!: CorpTypeCd
```

Comment:
I do this because I like to see the method signature and I don't like guessing where this reference actually exists (ie, in one of the mixins).